pkg/controller/eventing/subscription/reconcile.go

```diff
@@ -220,12 +219,12 @@ func (r *reconciler) resolveFromChannelable(namespace string, ref *corev1.Object
 	}
 
 	c := duckv1alpha1.Subscription{}
-	err = duck.FromUnstructured(*obj, &c)
+	err = duck.FromUnstructured(obj, &c)
 	return &c, err
 }
 
 // fetchObjectReference fetches an object based on ObjectReference.
-func (r *reconciler) fetchObjectReference(namespace string, ref *corev1.ObjectReference) (*unstructured.Unstructured, error) {
+func (r *reconciler) fetchObjectReference(namespace string, ref *corev1.ObjectReference) (duck.Marshalable, error) {
```

Comment:
I guess this change to the Unstructured support was brought in by the knative/pkg upgrade.

Reply:
pkg playing it fast and loose with compat :D